framework/Models/ROM.py

```diff
@@ -1400,6 +1405,20 @@ def getInitParams(self):
     paramDict = self.supervisedEngine.getInitParams()
     return paramDict
 
+  def provideExpectedMetaKeys(self):
+    """
+      Overrides the base class method to assure child engine is also polled for its keys.
+      @ In, None
+      @ Out, meta, tuple, (set(str),dict), expected keys (empty if none) and the indexes related to expected keys
```

Comment:
output docstring does not match the output

Reply:
fix: I will split over 2 entries for clarity.